tests/python/multidevice/test_transformer.py

```diff
@@ -349,24 +349,12 @@ def transformer_forward_definition(
     fd.add_output(out)
 
 
-def transformer_forward_multidevice_schedule(fd: FusionDefinition, num_devices: int):
+def transformer_forward_multidevice_schedule(
```

Comment:
With direct bindings, the schedule doesn't have to live in a separate function from the definition. This way, you can remove the input-unpacking code. (This could have been addressed when the test was migrated from legacy bindings to direct bindings, but I didn’t push for it at the time to keep the migration moving quickly.)